docs/README.md

```diff
@@ -52,67 +52,93 @@ You'll need a `kubectl`-style config file to connect to your cluster.
 `$HOME/.kube/config`. You can specify an alternate kubeconfig connection file
 with `--kubeconfig`, or the env var `$KUBECONFIG`, for any command.
 
-## Kn Config
+## Configuration
 
-There are a set of configuration parameters you can setup to better customize
+There are a set of configuration parameters you can set up to customize
 `kn`. For example, you can specify where your `kn` plugins are located and how
-they are found, and you can specify the prefix for your addressable `sink`
-objects. The `kn` configuration file is meant to capture these configuration
-options. Let's explore this file's location, and the options you are able to
-change with it.
+they are found or add prefixes for Knative eventing sink mappings.
 
 ### Location
 
-The default location `kn` looks for config is under the home directory of the
-user at `$HOME/.config/kn/config.yaml`. It is not created for you as part of the
-`kn` installation. You can create this file elsewhere and use the `--config`
-flag to specify its path.
+You can customize your `kn` CLI setup by creating a `config.yaml` configuration file. You can either provide the configuration via a `--config` flag or is picked up from a default location. The default configuration location conforms to the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and is different for Unix systems and Windows systems. If `XDG_CONFIG_HOME` env var is set, the default config location `kn` looks for is `$XDG_CONFIG_HOME/kn`, otherwise the file is looked up under the home directory of the user at `$HOME/.config/kn/config.yaml`. For Windows systems, the default `kn` configuration location is `%APPDATA%\kn`.
+
+`kn` does not create a default configuration file, nor does it write into an existing configuration file.
```

Comment:
Should we mention there that while we don’t create a default config, we do have defaults for everything in the config?

Reply:
Clarified it a bit.